src/rcl_logging_syslog.cpp

```diff
@@ -12,62 +12,64 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <stdarg.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <stdarg.h>
 #include <syslog.h>
 
 #include <memory>
 #include <system_error>
 
-#include <rcl_logging_interface/rcl_logging_interface.h>
-
-#include <rcpputils/env.hpp>
-#include <rcpputils/scope_exit.hpp>
+#include "rcpputils/env.hpp"
+#include "rcpputils/scope_exit.hpp"
 
-#include <rcutils/allocator.h>
-#include <rcutils/logging.h>
+#include "rcutils/allocator.h"
+#include "rcutils/logging.h"
 #include "rcutils/logging_macros.h"
 #include "rcutils/process.h"
-#include <rcutils/snprintf.h>
-#include <rcutils/strdup.h>
-#include <rcutils/time.h>
+#include "rcutils/snprintf.h"
+#include "rcutils/strdup.h"
+#include "rcutils/time.h"
+
+#include "rcl_logging_interface/rcl_logging_interface.h"
 
 static const char * facility_env_name = "RCL_LOGGING_SYSLOG_FACILITY";
 
 // see https://codebrowser.dev/glibc/glibc/misc/syslog.c.html#LogTag
 // This memory needs to be kept until closelog()
 static std::shared_ptr<std::string> syslog_identity;
 
-static const char *logger_name = "rcl_logging_syslog";
+static const char * logger_name = "rcl_logging_syslog";
```

Comment:
ubuntu 24.04 with jazzy and rolling, it does not cause the uncrustify warning something like this, but iron does.